src/pages/ServiceList/ItemDescription.tsx

```diff
@@ -49,24 +52,33 @@ export default class ItemDescription extends React.PureComponent<Props, State> {
   }
 
   render() {
+    var columnSize = 12 / this.state.nbColumns;
     return this.state.health ? (
       <Row>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={columnSize} lg={columnSize}>
           <strong>Health: </strong>
           <HealthIndicator id={this.props.item.name} health={this.state.health} mode={DisplayMode.SMALL} />
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={columnSize} lg={columnSize}>
           {!this.props.item.istioSidecar && <MissingSidecar />}
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={columnSize} lg={columnSize}>
           <strong>Config: </strong>{' '}
           <ConfigIndicator
             id={this.props.item.name + '-config-validation'}
             validations={[this.props.item.validation]}
             size="medium"
           />
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4} />
+        <Col xs={12} sm={12} md={columnSize} lg={columnSize}>
+          // If no api, the column is empty and not displayed
```

Comment:
these comments are displayed on screen :)
they should either be moved inside the `{...}` block or commented as html `<!-- -->`

Reply:
Wooo, how come I don't see it ? Let me change it

Reply:
Fixed